src/openrct2/editor.c

```diff
@@ -528,3 +529,174 @@ bool editor_check_park()
 
 	return true;
 }
+
+void game_command_edit_scenario_options(sint32* eax, sint32* ebx, sint32* ecx, sint32* edx, sint32* esi, sint32* edi, sint32* ebp)
+{
+	if (*ebx & GAME_COMMAND_FLAG_APPLY) {
```

Comment:
I've said this before but the game command should call a separate function with the registers converted to named parameters. This eases reading and makes it much easier to see the parameters of a game command.